tests/test_server.py

```diff
@@ -232,3 +236,34 @@ async def assessed_function(ctx: Context, param: str) -> str:
         result = await client.call_tool('assessed_function', {'param': 'value'})
         assert isinstance(result.content[0], TextContent)
         assert result.content[0].text == 'value'
+
+
+def test_json_logging(mocker):
+    log_config_file = Path(__file__).parent.parent / 'logging-json.conf'
+    assert log_config_file.is_file(), f'No logging config file found at {log_config_file.absolute()}'
+
+    # start the MCP server process with json logging
+    p = subprocess.Popen(
```

Comment:
Just FYI, there is also an async way for working with subprocesses

https://docs.python.org/3.12/library/asyncio-subprocess.html#asyncio-subprocess